### PR TITLE
Fix some bugs

### PR DESCRIPTION
**Pull request by cindytung**
- fix a section headers bug where all of them turned blue on hover, instead of just the schedule section headers
- disabled text select on schedule section headers
- change about me link to about

### CODE REVIEW

File: css/schedule.scss [hunk 1]
```diff
@@ -4,12 +4,18 @@
...
   margin-bottom: 20px;
   margin-top: 20px;
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
+  -khtml-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
 
   &:hover {
```
Comment (jubearsun) [inline css/schedule.scss:20]: can you make this cursor: pointer on hover? and have some transition (maybe 250ms)

**Review by jubearsun — CHANGES_REQUESTED** [review]

File: css/schedule.scss (hunk 1 above)
Comment (cindytung) [reply css/schedule.scss:20]: im using display:none to hide the result info on toggle, but is there a way to have add transition to display properties? or do i need to use max-height?

Comment (jubearsun) [reply css/schedule.scss:20]: Ah got it, you can't do transition on display but you can set opacity and height to 0 and keep display block

Comment (cindytung) [reply css/schedule.scss:20]: i fixed it but it actually looks a bit weird hmm

File: css/schedule.scss [hunk 2]
```diff
@@ -19,13 +19,22 @@
...
   }
 }
 
 .current__container, .fa15__container, .sp15__container {
   &.current--close, &.fa15--close, &.sp15--close  {
     .event__header, .result__content {
-      display: none;
+      max-height: 0px;
+      opacity: 0;
+      transition: max-height 2s;
```
Comment (jubearsun) [inline css/schedule.scss]: 2s is too long, can you change it to 250ms

**Review by jubearsun — CHANGES_REQUESTED** [review]

**Comment by cindytung** [comment]
SRY FOR THE LATE CHANGES HAHA

**Pull request merged**